nautilus_trader/adapters/databento/data.py

```diff
@@ -392,7 +392,7 @@ async def _subscribe_imbalance(self, data_type: DataType) -> None:
             live_client = self._get_live_client(dataset)
             live_client.subscribe(
                 schema=DatabentoSchema.IMBALANCE.value,
-                symbols=[instrument_id.symbol.value],
+                instrument_ids=[instrument_id],
```

Comment:
We also need to convert these instrument IDs to the `nautilus_pyo3.InstrumentId` version.